src/Plugin/Condition/AdoType.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Drupal\format_strawberryfield\Plugin\Condition;
+
+use Drupal\Core\Condition\ConditionPluginBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Url;
+use Drupal\strawberryfield\StrawberryfieldUtilityService;
+use Drupal\webform\Entity\WebformOptions;
```

Comment:
@patdunlavey ups. This one is no longer needed!

Reply:
Right! Fixed.